cmd/root.go

```diff
@@ -0,0 +1,32 @@
+// Package cmd for parsing command line arguments
+package cmd
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "tf-grid",
+	Short: "A cli for interacting with Threefold Grid",
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func init() {
+	log.Logger = zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr}).Level(zerolog.InfoLevel).
```

Comment:
we can use `zerolog.SetGlobalLevel(zerolog.InfoLevel)` and I think a debug flad would be great